### PR TITLE
update dlg loss function

### PR DESCRIPTION
**Pull request by wnma3mz**
Thank you for your work. It inspired me a lot.
I found that DLG loss function is problematic, maybe you can compare [utils.py](https://github.com/mit-han-lab/dlg/blob/master/utils.py)

```python
def cross_entropy_for_onehot(pred, target):
    return torch.mean(torch.sum(- target * F.log_softmax(pred, dim=-1), 1))
```

### CODE REVIEW

**Comment by PatrickZH** [comment]
Thanks for your suggestion. Your modification leads to the same results as us. So, no modification is needed.
